crates/runc/src/io.rs

```diff
@@ -261,9 +239,82 @@ impl Io for NullIo {
         Ok(())
     }
 
-    /// closing only write side (should be stdout/err "from" runc process)
```

Comment:
Not directly related to these changes, but do we need `close_after_start` ?
(Rust generally does Drop instead of Close).
Should we clean this as well?

Reply:
I feel it's still needed:)
Actually it reminds me that we need `impl<T: Io> Io for Arc<T>`.
The caller prepares an Arc<Io> object and passes a clone of it to Runc::create().
Runc::create() should call close_after_start() to release the writer side.
The caller could receive data from the Arc<Io> object then.